public/style.css

```diff
@@ -28,7 +28,7 @@ body {
 }
 
 h1 {
-  font-size: 2.5rem;
+  font-size: 28px;
```

Comment:
Not a big thing but in my opinion, it's generally preferable to use relative units, such as rem, over absolute inits, such as pixels for font sizes. The reason for this is that the font will then scale automatically with the device/browser preferences, the screen size, etc. 
I'd leave using pixels for visual components that should have the same size regardless of the screen size or configuration on the end device.